server.go

```diff
@@ -8,20 +8,26 @@ import (
 	"net/http"
 	"bytes"
 	"strings"
-
-	"github.com/clarifai/clarifai-go"
 	"github.com/gin-gonic/gin"
+	"os"
+	"net/url"
 )
 
 const (
 	clarifaiApi = "https://api.clarifai.com/v2/models/aaa03c23b3724a16a56b629203edc62c/outputs"
+	clarifyAuth = "https://api.clarifai.com/v1/token"
 )
 
 type Server struct {
 	app    *gin.Engine
-	client *clarifai.Client
 }
 
+type AccessToken struct {
+	Token string `json:"access_token"`
+	ExpiresIn int `json:"expires_in"`
```

Comment:
`ExpiresIn` is not used? It seems like each time we make a call to `clarifaiApi` we will use the client credentials to generate new access_token. So the token is just a one-time thing. I think just the Token is enough for us